server/routes/search.ts

```diff
@@ -53,6 +55,8 @@ export default function routes(
     const realNumberOfResults = errors.length > 0 ? 0 : numberOfResults
     const currentPageMax = parsedPage * pageSize
     const to = realNumberOfResults < currentPageMax ? realNumberOfResults : currentPageMax
+    auditService.prisonerSearch(search, 'HEI', res.locals.user?.username)
```

Comment:
Is it worth adding to the test for this route a check that audit called?

Reply:
Will have a nosey!